model/src/main/kotlin/VcsInfo.kt

```diff
@@ -105,8 +105,7 @@ fun VcsInfo?.orEmpty(): VcsInfo = this ?: VcsInfo.EMPTY
 private class VcsInfoDeserializer : StdDeserializer<VcsInfo>(VcsInfo::class.java) {
     companion object {
         val KNOWN_FIELDS by lazy {
-            VcsInfo::class.memberProperties.map { PROPERTY_NAMING_STRATEGY.translate(it.name) } +
-                    PROPERTY_NAMING_STRATEGY.translate("resolvedRevision")
+            VcsInfo::class.memberProperties.map { PROPERTY_NAMING_STRATEGY.translate(it.name) }
```

Comment:
BTW, do you remember why we introduced the check for unknown properties via `KNOWN_FIELDS`? Isn't it Jackson's default to fail on unknown properties if `@JsonIgnoreProperties(ignoreUnknown = true)` is not specified?

Reply:
I just verified that the `"fail if the input contains unknown fields"` test actually passes even if the `VcsInfoDeserializer` is removed. So maybe you could remove it completely as part of this PR?